arm/Microsoft.Storage/storageAccounts/deploy.bicep

```diff
@@ -152,7 +152,7 @@ var diagnosticsMetrics = [for metric in diagnosticMetricsToEnable: {
 }]
 
 var maxNameLength = 24
-var uniqueStorageNameUntrim = '${uniqueString('Storage Account${basetime}')}'
+var uniqueStorageNameUntrim = uniqueString('Storage Account${basetime}')
 var uniqueStorageName = length(uniqueStorageNameUntrim) > maxNameLength ? substring(uniqueStorageNameUntrim, 0, maxNameLength) : uniqueStorageNameUntrim
```

Comment:
```suggestion
var uniqueStorageName = take(uniqueStorageNameUntrim, maxNameLength)
```

Reply:
Suggesting to do this in a separate PR ;) Especially as we may ideally ditch the feature alltogether (see #1443)